tools/make_patch.c

```diff
@@ -681,13 +688,16 @@ struct patch *process_template(FILE *file, FILE *new_rom, FILE *orig_rom, FILE *
 						if (*p == ' ') *p = '_';
 					}
 
-					// Look for the symbol
-					symbol = find_symbol(symbols, buffer);
-					if (!symbol) {
-						fprintf(stderr, "Error: Cannot find symbol: %s\n",
-						        buffer);
+					// Look for the symbol starting with prefix
+					char *searchlabel = malloc(strlen(prefix) + strlen(buffer) + 1);
+					strcpy(searchlabel, prefix);
+					strcat((char *)searchlabel, buffer);
```

Comment:
Why the cast?

Reply:
Removed in https://github.com/pret/pokecrystal/pull/882/commits/bf8178e7328da3240535ef9800cdb5163466bcde